content/blog/Announcing_Tensor_Flow_as_the_Platinum_sponsor_Pycon_2017.md

```diff
@@ -0,0 +1,9 @@
+Title: Announcing Tensor Flow as Platinum Sponsor
+Date: 2017-10-25 12:40
+Category: 2017
+Slug: announcing-tensorflow-platinum-sponsor
+Summary: Announcing Tensor Flow as the Platinum Sponsor- Pycon 2017.
+
+<img src="https://lh3.googleusercontent.com/N_Nk4NMg4L3_1o8bj1eZR53rigiJXXkt34APyPBqb_gU3WmpCCyG5ArT69qkC80wNtuSHyUImM6R5fVpm_jWjSORekbJJkA=s688">
```

Comment:
Could you please shift the image after the first paragraph of the blog. Our blog feed will pick up an unpleasant scaled image for the in.pycon.org otherwise.